configure.ac

```diff
@@ -1375,13 +1375,21 @@ dnl ***************************************************************************
 if test "x$with_nosetests" != "x"; then
     NOSETESTS=$with_nosetests
 else
-    AC_CHECK_PROG(NOSETESTS, nosetests, nosetests)
+    if (echo "$with_python" | grep -Eq "python-?3.*"); then
```

Comment:
[optional] The same *grep* is used to create the `ENABLE_PYTHONv2/3`, I think it would be beneficial if the *magic* would be in one place, and we have `enable_pythonv2/3` variable in configure  (or `python_version == 2/3`).

Reply:
Good idea. I will do so.

Reply:
I just force pushed the correction. During teh fix I saw another problem: nosetests/pylint detection was still executed even if python was disabled. I fixed this by checking the `$python_found` variable.

Reply:
Thanks.